core/src/main/scala/org/apache/spark/security/SocketAuthHelper.scala

```diff
@@ -115,3 +115,19 @@ private[spark] class SocketAuthHelper(conf: SparkConf) {
   }
 
 }
+
+private[spark] object SocketAuthHelper {
+  def serveToStream(
+      threadName: String, authHelper: SocketAuthHelper)(writeFunc: OutputStream => Unit)
+    : Array[Any] = {
```

Comment:
nit: I guess I got this wrong before, but I think the line-breaking & indentation here should be more like `SocketAuthServer. setupOneConnectionServer`